### PR TITLE
add --show-ping to display ping, disabling it by default

### PR DESCRIPTION
**Pull request by towc**
builds on #110 

`--show-ping` still uses the `-P` shortcut, which might be confusing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Options:
   -o, --origin <origin>               optional origin
   -x, --execute <command>             execute command after connecting
   -w, --wait <seconds>                wait given seconds after executing command
+  -P, --show-ping                     print internal ping/pong messages
```
Comment (lpinca) [inline README.md]:
```suggestion
  -P, --show-ping                     print a notification when a ping or pong is received
```

File: bin/wscat [hunk 2]
```diff
@@ -116,6 +116,7 @@ program
   .option('-o, --origin <origin>', 'optional origin')
   .option('-x, --execute <command>', 'execute command after connecting')
   .option('-w, --wait <seconds>', 'wait given seconds after executing command')
+  .option('-P, --show-ping', 'print internal ping/pong messages')
```
Comment (lpinca) [inline bin/wscat]:
```suggestion
  .option(
    '-P, --show-ping',
    'print a notification when a ping or pong is received'
  )
```

**Comment by lpinca** [comment]
Not sure if `--show-ping` is a good name for the option as it also handles pongs but I can't think of anything better.

**Comment by towc** [comment]
maybe `--ping-pong`?

**Comment by lpinca** [comment]
Yes sounds like a better name.

**Comment by lpinca** [comment]
Thanks.

**Pull request merged**